src/Features/VisualBasic/Portable/InlineHints/VisualBasicInlineParameterNameHintsService.vb

```diff
@@ -96,5 +96,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.InlineHints
             Dim propertySymbol = TryCast(parameter.ContainingSymbol, IPropertySymbol)
             Return propertySymbol IsNot Nothing AndAlso propertySymbol.IsDefault
         End Function
+
+        Protected Overrides Function HintText(parameterName As String) As String
```

Comment:
is there a way to test that the hint text is correct on VB?

Reply:
yea I'm going through and changing the tests